enhancements/builds/entitlements.md

```diff
@@ -44,151 +44,95 @@ see-also:
->availability of such a feature (probably "not sure")?  Upstream feature work could be implied with such an item.  But
->what influence such a notion could have on this proposal is TBD. 
-> 3. After 4.5 epic planning it was made clear that a more general solution around entitlements, beyond just builds,
->is desired.  PM has the to-do to investigate, prioritize, and create/assign epics accordingly.  Investigating in the 
->interim around build specific needs via this proposal has been deemed acceptable, but certainly a subsequent, broader
->proposal may necessitate adjustments to this one.  This proposal at the moment is straddling the fence between speaking
->to a generic solution vs. a build specific one (though things are trending toward the former).  But a rebranding of 
->this proposal may occur during the review process.
-> 4. We did *NOT* provide for multiple sets of entitlement credentials in 3.x.  There have been no official, RFE style  
->registered requirements entitlements at a namespace level.  However, we've started to get (slightly) less official
->feedback that customers want this in a more automated sense (vs. the current manual steps).  At a minimum, and is
->noted below when alternatives are discussed, if namespaced entitlements are added with subsequent implementation
->after global entitlements usability is improved, it needs to be seamless, and we minimally need to agree now
->on what the order of precedence will be (with the take on that listed below).
-> 5. There has also been a question in the past needing to shard entitlements due to API rate limiting.  To date, it
->has been stated that is not a concern for builds.  Periodic checkpoints here could be prudent.
-> 6. There is also the general question of how entitlements are propagated for our managed service offerings.  There are
-some attempts at capturing the particulars for this below, but it needs some vetting.
-> 7. With the CSI plugin based solution articulated below, there is some question as to how that is packaged, such as a)
-> a separate admission server (possibly aggregated apiserver) utilizing
->libraries like https://github.com/openshift/generic-admission-server, or b) perhaps even a patch we carry on the k8s
->apiserver (yikes!!)
+1. We did *NOT* provide for multiple sets of entitlement credentials in 3.x.  There have been no official, RFE style registered requirements entitlements at a namespace level.  However, we've started to get (slightly) less official feedback that customers want this in a more automated sense (vs. the current manual steps).  At a minimum, and is noted below when alternatives are discussed, if namespaced entitlements are added with subsequent implementation after global entitlements usability is improved, it needs to be seamless, and we minimally need to agree now on what the order of precedence will be (with the take on that listed below).
```

Comment:
```suggestion
1. We did *NOT* provide for multiple sets of entitlement credentials in 3.x.  There have been no official, RFE style registered requirements entitlements at a namespace level.  However, we've started to get (slightly) less official feedback that customers want this in a more automated sense (vs. the current manual steps).  At a minimum, and is noted below when alternatives are discussed, if namespaced entitlements are added with a subsequent implementation after global entitlements usability is improved, it needs to be seamless, and we minimally need to agree now on what the order of precedence will be (with the take on that listed below).
```

Reply:
in case it is not clear, I change `with subsequent implementation` to `with a subsequent implementation`